### ISSUE TITLE
iPod in Use

### ISSUE THREAD

**Issue by spacequeer** [cormiertyshawn895/Retroactive#11]
Hi! New to Github and this—loving what you've done with iTunes. My problem is that I wanted to use iTunes to manage an older iPod nano instead of Finder, but whenever I try to use iTunes to restore that iPod, I receive an error, and can only restore it with Finder. In addition, iTunes gives me an 'iPod in Use' error when I try to sync it—it says 'This iPod is in use by another user on this Mac." Is there a workaround for this, or maybe to have Finder not recognize the iPod?

**Comment by cormiertyshawn895** [comment]
What happens if you open iTunes first, and then connect your iPod nano when iTunes is open? Does Finder still take control of your iPod before iTunes?

**Comment by spacequeer** [comment]
Finder still takes control and iTunes shows up with the message 'This iPod is in use by another user on this Mac'

**Comment by jcohen-lml** [comment]
I have exactly the same problem

**Comment by count0x30** [comment]
Happening for me with a 4th gen iPod shuffle.  Two iPhones and an iPad mini show up for me under Devices in iTunes, and I can manage them and have successfully synced and made backups.  But when I plug in the shuffle, along with the message that it's in use by another user, iTunes tells me:
> iTunes cannot read the contents of the iPod "IPOD". Go to the Summary tab in iPod preferences and click Restore to restore this iPod to factory settings.

When I do that, I get the error:
> The iPod "iPod" could not be restored. An unknown error occurred (1433).

I tried installing iTunesAccess with [these instructions](https://forums.macrumors.com/threads/has-anyone-gotten-an-ipod-shuffle-2nd-gen-to-work.2196567/?post=27942067#post-27942067), but nothing changed.

**Comment by panahead** [comment]
I am not sure how to reply or to ask questions re Retroactive… But firstly thank you for creating it! I am using it to have the ios remote app work on itunes… to be able to adjust the volume levels on my various airplay devices in my home… and it is working great! thank you! my question is is it bad to have both itunes and the Music apps both open on my desktop at the same time? should I just use itunes only now? I have over 175 gigs of music on my hard drive and I also subscribe to iTunes Match… so the 4 or 5 computers plus the ipads and iphones i have spread all over the state all have the same library and playlists…

will this work for me?

Thanks for your time and patience…

Josh

**Comment by Antocratie** [comment]
Hi, thank you very much for your work, same issue, is there a solution to fix this ? Maybe uninstalled the main Music App ? Regards,

**Comment by tghuguenin** [comment]
Same issue here with an iPod classic 5th gen. I was told that itunes would work the same as before. It doesn't do the main two things that I need it for (ripping an audio CD and syncing music+podcasts to ipod). So it is basically useless until someone finds out a way to fix this.

**Comment by sztenni** [comment]
Same issue here. I opened another thread on this before finding this one. iPod Classic 5th generation.

**Comment by sztenni** [comment]
Update: no more alert about another user, but iTunes asks me to reset iPod...

**Comment by cormiertyshawn895** [comment]
### If you see "The iPod is in use by another user on this computer", or nothing happens at all when you plug your iPod into iTunes:
<img width="602" alt="in-use" src="https://user-images.githubusercontent.com/57056759/75617416-aa6d0880-5b13-11ea-9023-622193684295.png">

- Disconnect the device from your Mac, but keep iTunes open.
- Click on the Spotlight icon (magnifying glass) on the menu bar.
- Type "Terminal" and press return to open the Terminal app.
- Enter `killall AMPDevicesAgent` in the Terminal window, and press return.
- Connect the Apple device back to your Mac.

---
### If you see "iTunes cannot read the contents of the iPod. Go to the Summary tab in iPod preferences and click Restore to restore this iPod to factory settings":
<img width="602" alt="cannot-read" src="https://user-images.githubusercontent.com/57056759/75617419-ae008f80-5b13-11ea-85e7-17ffb4bc907a.png">

- Re-install iTunes [using the latest version of Retroactive](https://github.com/cormiertyshawn895/Retroactive/releases/).
- After re-installing iTunes, Retroactive will ask you about iPod syncing. Click "Enable Full Disk Access", and follow on-screen instructions provided by Retroactive.

![7a](https://user-images.githubusercontent.com/57056759/75648829-2182c980-5c06-11ea-8ee7-7a8a0eacfd2f.jpg)

---
### If you try to enable VoiceOver for iPod shuffle, but see "iTunes was unable to install the VoiceOver Kit. An unknown error occurred (1701)":

- [Directly download the VoiceOver 1.4.2 installer here](https://swdist.apple.com/content/downloads/29/37/041-91732-A_LSWLP6NLRV/bk8l36k29x8d146doiwcvuv8qemu3mzxwk/VoiceOver.pkg).
- Install the downloaded VoiceOver.pkg.
- Quit and re-open iTunes.

**Comment by Neucleus** [comment]
Catalina 10.15.5
iTunes 12.9.5.5
Retroactive 1.8
**Message**: `This iPhone is in use by another user on this Mac`
I know this has been discussed a lot in this forum, but I think I've tried everything. My iPod works in iTunes just fine. When I connect my iPhone XS, I get this message. I can see my iPhone in iTunes and I can view all the playlists and songs, but I can't drag/drop any songs in to it from iTunes.

What I've tried:
- Reinstalled iTunes 1.8
- Enabled full disk access for iTunes and for the BASH script during Retro/iTunes install. Confirmed settings saved
- Finder / Preferences > Locations : CDs, DVD, and iOS Devices (checked)
- Restarted Mac
- Open iTunes. Ran `killall AMPDevicesAgent` from Terminal. Plugged in iPhone
iPhone isn't showing in Finder. Shows in iTunes as view only. I can't add any songs to it
